src/Octoshift/Services/GithubApi.cs

```diff
@@ -206,7 +204,7 @@ public virtual async Task<string> GetEnterpriseId(string enterpriseName)
         }
     }
 
-    public virtual async Task<string> CreateAdoMigrationSource(string orgId, string adoServerUrl)
+    public virtual async Task<string> CreateAdoMigrationSource(string orgId, string orgName, string adoServerUrl)
```

Comment:
It's definitely a bit of a smell that we have to pass the `orgName` to this method in order to generate the nice error message.

I would be open to refactoring this so that `GithubApi` throws an exception with the raw error message from the API, and then the caller is responsible for catching that error and decorating it. That would work - the only disadvantage is that this class is easier to unit test that the command handler.

Reply:
As discussed below, we won't be able to make GraphQL return the org name because this is an error response where we don't get to control the fields.